lib/provider/kleinanzeigen.js

```diff
@@ -25,6 +25,7 @@ const config = {
     link: '.aditem-main .text-module-begin a@href | removeNewline | trim',
     description: '.aditem-main p:not(.text-module-end) | removeNewline | trim',
     address: '.aditem-main--top--left | trim | removeNewline',
+    image: '.aditem-image .imagebox img@src',
```

Comment:
Would you mind adding this for the other providers as well, at least where it is available?